definitions/region/README.md

```diff
@@ -142,6 +142,22 @@ Example:
 
 > Norway>Germany
 
+To represent data that refers to a flow or cost contribution of a region
+to a connection between two regions, the contributing region name 
+can be combined with the bi-directional data using a `:` character before
+the bi-directional data (without spaces before/after that character).
+
+Example:
+
+> Germany:France>Spain
+
+The contributing region can be of **a different level of spatial detail**
+from the two regions specified in the bi-directional data.
+
+Example:
+
+> Berlin:France>Spain
```

Comment:
Note that "Berlin" is not a region-name defined in this project, I'd rather use the ISO3-code "DE300", right?
```suggestion
> Berlin:France>Spain
```